improved code for instabot part 1 selenium.ipynb as the structure of the website has changed a little

vatsalyrai opened the pull request:
Instagram website' layout/structure has changed a little, so i have improved the login functionality and "turn-off notification " functionality  of the code

vatsalyrai committed:
Update project instabot part 1 selenium.ipynb

parikshit223933 commented:
Hey, @vatsalyrai,
I appritiate your effort in contributing to this repository, however I dont have the resources to test this particular merge request. can you please attach a video showing that the code changes you have done actually work? It will reduce a lot of un-necessary effort of mine to install and set up python and all the required development tools on my system. This project was done way back in the past and since then I have changed many things including the OS.

vatsalyrai commented:
Actually i have only fixed the login functionality and turn of
notifications stuff.
Some more functionalities in the code don't work , i can fix some of them
but not all .
I will send you the video tommorow of the functionality that i have changed

Regards
Thank you

On Thu, Oct 1, 2020, 6:10 PM Parikshit Singh <notifications@github.com>
wrote:

> Hey, @vatsalyrai <https://github.com/vatsalyrai>,
> I appritiate your effort in contributing to this repository, however I
> dont have the resources to test this particular merge request. can you
> please attach a video showing that the code changes you have done actually
> work? It will reduce a lot of un-necessary effort of mine to install and
> set up python and all the required development tools on my system. This
> project was done way back in the past and since then I have changed many
> things including the OS.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/parikshit223933/Coding-ninjas--Data-Science-and-Machine-Learning/pull/2#issuecomment-702106771>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARAFLYRFNED3KMFXRBUNWYDSIR2FBANCNFSM4R75L6EA>
> .
>

Utkarsh-jain8077 commented:
hey can u please tell us the solution of amazon dataset and terrorism dataset . kindly helpfull.
